app/views/users/dashboard.html.erb

```diff
@@ -0,0 +1,95 @@
+
+<h1> <%= @user %> </h1>
+
+<%= render "partials/dashboard/text_fields", value: "Lorem Ipsum of a big description of someone that has a lot to say about themselves bla bla bla bla bla bla yada yada" %>
+
+
+</div>
+<div class="flex flex-col bg-slate-300 ml-2 mr-2 rounded w-1/2 m-6 justify-center">
+
+<div class="flex flex-col bg-slate-300 ml-2 mr-2 rounded m-6 text-center">
+
+<%= render "partials/dashboard/h3", label: "X", value: @user_X %>
+<%= render "partials/dashboard/h3", label: "Instagram", value: @user_instagram %>
+<%= render "partials/dashboard/h3", label: "Pinterest", value: @user_pinterest %>
+</div>
+<div class="text-center">
+<h1> <%= @outfits_amount %> </h1>
+<h3> Pieces </h3>
+
+<h1> <%= @pieces_amount %> </h1>
+<h3> Outfits </h3>
+</div>
+</div>  
+
+
+
+</div>
+
+<h1> <%= @latest_clothes %> </h1>
+<h1> <%= @latest_outfits %> </h1>
+
+<div class="flex items-start w-full ml-6">
+<h1 class="text-white"> Latest clothes </h1>
+</div>
+
+
+<div class="flex w-full p-6 overflow-scroll">
+
+<%= render "partials/dashboard/piece", image_path: "test.jpg", alt_text: "Placeholder", name: "placeholder", category: "Punk lodsqjdsqjdqsjdqlds", size: "m -32" %>
+<%= render "partials/dashboard/piece", image_path: "test.jpg", alt_text: "Placeholder", name: "placeholder", category: "placeholder", size: "m -32" %>
+<%= render "partials/dashboard/piece", image_path: "test.jpg", alt_text: "Placeholder", name: "placeholder", category: "placeholder", size: "m -32" %>
+<%= render "partials/dashboard/piece", image_path: "test.jpg", alt_text: "Placeholder", name: "placeholder", category: "placeholder", size: "m -32" %>
+<%= render "partials/dashboard/piece", image_path: "test.jpg", alt_text: "Placeholder", name: "placeholder", category: "placeholder", size: "m -32" %>
+<%= render "partials/dashboard/piece", image_path: "test.jpg", alt_text: "Placeholder", name: "placeholder", category: "placeholder", size: "m -32" %>
+<%= render "partials/dashboard/piece", image_path: "test.jpg", alt_text: "Placeholder", name: "placeholder", category: "placeholder", size: "m -32" %>
+<%= render "partials/dashboard/piece", image_path: "test.jpg", alt_text: "Placeholder", name: "placeholder", category: "placeholder", size: "m -32" %>
+<%= render "partials/dashboard/piece", image_path: "test.jpg", alt_text: "Placeholder", name: "placeholder", category: "placeholder", size: "m -32" %>
+<%= render "partials/dashboard/piece", image_path: "test.jpg", alt_text: "Placeholder", name: "placeholder", category: "placeholder", size: "m -32" %>
+<%= render "partials/dashboard/piece", image_path: "test.jpg", alt_text: "Placeholder", name: "placeholder", category: "placeholder", size: "m -32" %>
+<%= render "partials/dashboard/piece", image_path: "test.jpg", alt_text: "Placeholder", name: "placeholder", category: "placeholder", size: "m -32" %>
+
+
+</div>
+<% @latest_clothes.each do | piece |%>
```

Comment:
Use the logic from here to render your "dashboard/piece" ->
<%= render "partials/dashboard/piece", image_path: "test.jpg", alt_text: piece.name, name: piece.name, category: "Punk lodsqjdsqjdqsjdqlds", size: "m -32" %>